reactioncommerce/reaction-component-library
# feat: Cart item read-only

Resolves #221
Impact: **minor**  
Type: **feature**

## Component

- CartItems: add `isReadOnly` prop that hides the remove button and quantity input of all cart items
- CartItem: add `isReadOnly` prop that hides the remove button and quantity input
- CartItemDetal: add a prop to display quantity as text

## Screenshots

<img width="567" alt="cartitem_ _reaction_design_system" src="https://user-images.githubusercontent.com/42217/44125154-d47df55c-9fe5-11e8-9849-ede041e134ca.png">

## Breaking changes
none

## Testing

1. Open the docs to CartItems, CartItem, and CartItemDetail pages
2. See various states where the quantity input and remove button are not visible (isReadOnly example at the bottom of the CartItem and CartItems pages)
3. Verify that the quantity attribute shows up in the CartItemDetail example if a non-zero quantity is provided
